easybuild/easyconfigs/s/SHTns/SHTns-3.5.1-foss-2021b.eb

```diff
@@ -8,14 +8,26 @@ description = """Spherical Harmonic Transform library aimed at high performance
 geometries."""
 
 toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
 
 source_urls = ['https://bitbucket.org/nschaeff/shtns/downloads/']
 sources = [SOURCELOWER_TAR_GZ]
 checksums = ['77f8a33b94df8786d2ce9b95cbfbe548f00443625b310b8c64012b22c8a7394f']
 
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+]
+
+installopts = " && pip install --no-deps --ignore-installed --prefix %(installdir)s ."
```

Comment:
What do you think about using an extension instead? Then you get all the goodies of `PythonPackage` and the python module can be searched as extension
```suggestion
# install Python bindings and its dependencies
exts_defaultclass = 'PythonPackage'
exts_default_options = {
    'download_dep_fail': True,
    'use_pip': True,
    'sanity_pip_check': True,
}
exts_list = [
    ('shtns', version, {
        'source_tmpl': SOURCELOWER_TAR_GZ,
        'checksums': ['77f8a33b94df8786d2ce9b95cbfbe548f00443625b310b8c64012b22c8a7394f'],
    }),
]
```

Reply:
makes total sense, this is significantly better (though less verbose), thanks @lexming!